<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="configTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="openglTab">
      <attribute name="title">
       <string>OpenGL</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="openglInfoBox">
         <property name="title">
          <string>OpenGL信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="glVendorLabel">
            <property name="text">
             <string>GL_VENDOR</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="glVendorText">
            <property name="text">
             <string>GL_VENDOR_TEXT</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="glVersionLabel">
            <property name="text">
             <string>GL_VERSION</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="glVersionText">
            <property name="text">
             <string>GL_VERSION_TEXT</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="glRendererLabel">
            <property name="text">
             <string>GL_RENDERER</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="glRendererText">
            <property name="text">
             <string>GL_RENDERER_TEXT</string>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="glExtensionsLabel">
            <property name="text">
             <string>GL_EXTENSIONS</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPlainTextEdit" name="glExtensionsText">
            <property name="documentTitle">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string>GL_EXTENSIONS_TEXT GL_EXTENSIONS_TEXT GL_EXTENSIONS_TEXT GL_EXTENSIONS_TEXT GL_EXTENSIONS_TEXT GL_EXTENSIONS_TEXT GL_EXTENSIONS_TEXT GL_EXTENSIONS_TEXT GL_EXTENSIONS_TEXT
</string>
            </property>
            <property name="placeholderText">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="renderSettingsBox">
         <property name="title">
          <string>渲染设置（重启程序生效）</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelCullFace">
              <property name="text">
               <string>表面剔除</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioCullCW">
              <property name="text">
               <string>逆时针</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">culLFaceButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioCullCCW">
              <property name="text">
               <string>顺时针</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">culLFaceButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioNoCull">
              <property name="text">
               <string>无</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">culLFaceButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelImageTexFilt">
              <property name="text">
               <string>图片纹理过滤</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioImgTexFiltNearest">
              <property name="text">
               <string>近似</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">imgTexFiltButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioImgTexFiltLinear">
              <property name="text">
               <string>线性</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">imgTexFiltButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioImgTexFiltMipmapNearest">
              <property name="text">
               <string>近似MipMap</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">imgTexFiltButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioImgTexFiltMipmapLinear">
              <property name="text">
               <string>线性MipMap</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">imgTexFiltButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="labelScreenTexFilt">
              <property name="text">
               <string>屏幕纹理过滤</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioScrTexFiltNearest">
              <property name="text">
               <string>近似</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">scrTexFiltButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioScrTexFiltLinear">
              <property name="text">
               <string>线性</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">scrTexFiltButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="screenRenderMode">
              <property name="text">
               <string>屏幕渲染模式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioRenderPreGaussian">
              <property name="text">
               <string>预渲染纹理</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">scrRenderModeButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioRenderRealtime">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>实时RTT</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">scrRenderModeButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cameraTab">
      <attribute name="title">
       <string>摄像机</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="faceAnimationTab">
      <attribute name="title">
       <string>屏幕动画</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="robotBoneTab">
      <attribute name="title">
       <string>骨骼动画</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="faceCaptureTab">
      <attribute name="title">
       <string>面部捕捉</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="soundAnalysisTab">
      <attribute name="title">
       <string>音频分析</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string>关于</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="imgTexFiltButtonGroup"/>
  <buttongroup name="culLFaceButtonGroup"/>
  <buttongroup name="scrTexFiltButtonGroup"/>
  <buttongroup name="scrRenderModeButtonGroup"/>
 </buttongroups>
</ui>
